src/yews/models/cpic.py

```diff
@@ -1,13 +1,14 @@
-# TO-DO: need to add model_zoo utility and pretrained models.
 import torch.nn as nn
 
 from .utils import load_state_dict_from_url
 
-__all__ = ['CpicV1', 'CpicV2', 'cpic_v1', 'cpic_v2']
+
+__all__ = ['CpicV1', 'cpic_v1', 'CpicV2', 'cpic_v2', 'CpicV3', 'cpic_v3']
 
 model_urls = {
     'cpic_v1': 'https://www.dropbox.com/s/ckb4glf35agi9xa/cpic_v1_wenchuan-bdd92da2.pth?dl=1',
-    'cpic_v2': 'https://www.dropbox.com/s/kyiuprnn8014fs5/cpic_v2_wenchuan-ee92060a.pth?dl=1'
+    'cpic_v2': 'https://www.dropbox.com/s/kyiuprnn8014fs5/cpic_v2_wenchuan-ee92060a.pth?dl=1',
+    'cpic_v3': 'xxxx'
```

Comment:
Are you planning to upload a pretrained model for cpic_v3? You can use `''` instead of `'xxxx'` for place holder.

Reply:
Resolve. Please check it again. Thanks.